Scripts/Tools/ResizeEditor.cs

```diff
@@ -292,6 +350,12 @@ void OnKeyAction(SceneView sceneView, Event e)
 
 			if(!CameraPanInProgress)
 			{
+                // check for vertex snapping in translate mode.
+                if (e.keyCode == KeyCode.V)
+                {
+                    vertexSnapping = e.type == EventType.KeyDown;
```

Comment:
Perhaps this is just personal preference but this kind of line reads more easily with brackets, i.e.

vertexSnapping = (e.type == EventType.KeyDown);

Reply:
I will keep that in mind, you are right. :)